js/src/tests/data/snapshot.ts

```diff
@@ -0,0 +1,768 @@
+import { PublicKey } from '@solana/web3.js';
+import BN from 'bn.js';
+import { Snapshot } from '../../types';
+
+export const snapshot: Snapshot = {
+  solido: {
+    lido_version: 0,
+    manager: new PublicKey(
+      new BN(
+        '7003585ceaf54c89ac039fa0f2bb390729c289e416b346306bf277ed4561c6e4',
+        'hex',
+        'le'
+      )
+    ),
+    st_sol_mint: new PublicKey(
+      new BN(
+        'fedef235d5df2a94343d6348287dbfc85f31ffc815d800ce9d6b471971cb7162',
+        'hex',
+        'le'
+      )
+    ),
+    exchange_rate: {
+      computed_in_epoch: new BN('0100', 'hex'),
+      st_sol_supply: new BN('35e06bf954dd5', 'hex'),
+      sol_balance: new BN('36a84a87b4459', 'hex'),
```

Comment:
This is fine, but it may be more readable to just use base 10 here, i.e.

```javascript
computed_in_epoch: new BN(256),
st_sol_supply: new BN(947_808_007_179_733),
sol_balance: new BN(961_542_925_010_009),
```

Or is this data generated and this is the default way in which it’s printed? If so, please add a comment to explain how to re-generate the data if needed.